data/coalition/coalition culture conversations.txt

```diff
@@ -233,3 +233,137 @@ mission "Coalition Folklore: Kimek"
+		not attributes "farming"
+	on offer
+		conversation
+			`The spaceport here on <origin> is in something of an uproar. Thousands of Kimek are seemingly organized in what, strangely enough for a world in the Coalition, resembles a protest. Carrying about some signs or devices repeating a message in the Kimek language, or an icon that depicts the usual Kimek diet, with their many crops sharing some space with a few fish. Strangest of all, however, is how the local Heliarchs are handling the situation: they appear almost hesitant, defensively speaking to the civilians in attempts to calm them down, sometimes apologizing or simply being forced to listen to the complaints as several Kimek pile up around them.`
+			`	When you finally manage to find a Heliarch who isn't occupied trying to keep the area in order, a Saryd, he explains the situation. "Missed its delivery, a food convoy has. Still investigating the issue, we are; know only that, somehow, gone missing, the entire fleet has..." He sighs, shaking his head. "Keep the locals calm we must." He goes on to help the other Heliarchs, without another word.`
+			`	Within a few minutes that must've felt like hours for the stationed Heliarch agents, battalions of drones start making their way from the tallest building in the city - the hunger tower. Ferrying around several meals each, with impressive precision, the drones fly throughout the civilian mass, distributing the meals. Hundreds of larger head to the residential areas of the city, presumably to provide the necessary supplies for the citizens until the next food convoy arrives. Seemingly satisfied, for the moment at least, the Kimek gradually dissipate out of the spaceport and back to their homes, leaving the spaceport to the now exhausted Heliarch agents.`
+				decline
+
+
+
+mission "Coalition Incident: Arach Store"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		random < 2
+	source
+		government "Coalition"
+		attributes "arach"
+		not attributes "coalition station"
+		not planet "Ahr"
+	on offer
+		conversation
+			`A Heliarch patrol seems to be discussing with an Arach shop worker in front of a large, prestigious looking Arach clothing store in the spaceport. Other agents prevent you and other bystanders from approaching the scene, but you can still make out some context from their gesturing. It seems the Heliarchs are there to confiscate some pieces from the shop that you guess don't abide by their laws, while the shop worker keeps worryingly attempting to answer what questions they pose him.`
+			`	Eventually, the shop owner arrives. Judging by her fine clothes, and from the reactions of the local Arachi, you figure she must either be really famous here on <origin>, or possibly belong to one of the Arach Houses, though nothing on her attire seems to indicate any specific house. She tranquilizes the worker, and takes on to speaking with the Heliarch agents as he heads back inside.`
```

Comment:
> She tranquilizes the worker

Uhh... what? The boss just walks up to him and shoots/drugs him?

Reply:
It would appear that is not a very good direct translation from portuguese. Whoopsie. Changed it.